components/Products.css

```diff
@@ -0,0 +1,171 @@
+.main-container {
+  display: flex;
+}
+.filters {
+  background-color: var(--primary-dark-color);
```

Comment:
Trry to add a little margin left in filter section

Reply:
Okayy!! Will do that.